JitsiConference.js

```diff
@@ -1453,14 +1449,14 @@ JitsiConference.prototype._acceptJvbIncomingCall = function(
 
     // Log "session.restart"
     if (this.wasStopped) {
-        Statistics.sendEventToAll(SESSION_RESTART);
+        Statistics.sendEventToAll(createJingleEvent('restart'));
     }
 
-    Statistics.analytics.sendEvent(
-        SESSION_INITIATE,
+    Statistics.analytics.sendEvent(createJingleEvent(
+        'session-initiate.received',
         {
-            value: now - this.room.connectionTimes['muc.joined']
-        });
+            value: now
```

Comment:
Why the value change?

Reply:
To be consistent with the rest of the events. Eventually I think this should be moved to a separate module together with all the messy "connectionTimes" logic